test/functional/test_ivn.py

```diff
@@ -93,6 +96,7 @@ def _start_node(self, node_info):
         fake_node_obj.init()
         fake_node_obj.precheck()
         fake_node_obj.start()
+        fake_node_obj.wait_node_up(timeout=10)
         return fake_node_obj
 
     def node_config(self, node_name, ns_name, boot_image):
```

Comment:
better to change to "start_node_with_config" because it starts node at the end, not just config node.